public/app/features/alerting/unified/components/notification-policies/Policy.tsx

```diff
@@ -742,7 +742,7 @@ const TimeIntervals: FC<{ timings: string[]; alertManagerSourceName: string }> =
               href={createMuteTimingLink(timing, alertManagerSourceName)}
               color={canSeeMuteTimings ? 'primary' : 'secondary'}
               variant="bodySmall"
-              inline={false}
+              inline={canSeeMuteTimings ? false : undefined}
```

Comment:
This was just giving a console error for incorrectly passing `true` to a prop that didn't support it (`Text` didn't let me pass the prop, but `TextLink` does)